code/game/objects/items/mail.dm

```diff
@@ -140,6 +140,8 @@
 
 /// Accepts a mind to initialize goodies for a piece of mail.
 /obj/item/mail/proc/initialize_for_recipient(datum/mind/recipient)
+	// PLEASE, IF YOU GONNA CHANGE HERE SOMETHING, CHECK FOR BUGS WITH TRAITOR MAILS!!
```

Comment:
What kind of bugs are you anticipating? Comments like these do not do anything and are not worth having. If there is a reasonably possible bug then it should be unit tested.

Reply:
i will add unit test for that, yeah.